### PR TITLE
EggLauncher

### PR DESCRIPTION
**Pull request by harliq**
3 weenies for Egg Launcher
04078 is spell
35179 is spell projectile
35188 is Egg Launcher

### CODE REVIEW

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+VALUES (35179,   1, True ) /* Stuck */
+     , (35179,  14, False) /* GravityStatus */
+     , (35179,  15, True ) /* LightsStatus */
+     , (35179,  16, True ) /* ScriptedCollision */
+     , (35179,  17, True ) /* Inelastic */
+     , (35179,  24, True ) /* UiHidden */;
+
+INSERT INTO `weenie_properties_float` (`object_Id`, `type`, `value`)
+VALUES (35179,  39,     0.9) /* DefaultScale */
+	 , (35179,  26,      15) /* MaximumVelocity */
```
Comment (DridsGhost) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql:23]: please fix formatting

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+     , (35188, 144,       0) /* ManaConversionMod */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (35188,   1, 'Egg Launcher') /* Name */
+     , (35188,  16, 'A nervous hen.') /* LongDesc */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (35188,    1,   33560288) /* Setup */
+     , (35188,    3,  536870932) /* SoundTable */
+     , (35188,    6,   67110219) /* PaletteBase */
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:38]:
```suggestion
     , (35188,    6,   67110219) /* PaletteBase */
     , (35188,    7,  268436660) /* ClothingBase */
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+     , (35188,    8,  100674625) /* Icon */
+     , (35188,   22,  872415275) /* PhysicsEffectTable */
+     , (35188,   27, 1090519043) /* UseUserAnimation - UseMagicWand */
+     , (35188,   28,       4078) /* Spell - EggBolt */;
+
+
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:44]: delete everything after this line
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (35188,   1,      32768) /* ItemType - Caster */
+     , (35188,   5,         30) /* EncumbranceVal */
+     , (35188,   9,   16777216) /* ValidLocations - Held */
+     , (35188,  10,   16777216) /* CurrentWieldedLocation - Held */
+     , (35188,  16,    6291460) /* ItemUseable - SourceWieldedTargetRemoteNeverWalk */
+     , (35188,  18,          1) /* UiEffects - Magical */
+     , (35188,  19,       2300) /* Value */
+     , (35188,  33,          1) /* Bonded - Bonded */
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:14]:
```suggestion
     , (35188,  33,          1) /* Bonded - Bonded */
     , (35188,  46,        512) /* DefaultCombatStyle - Magic */
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 5]
```diff
@@ -0,0 +1,49 @@
+DELETE FROM `weenie` WHERE `class_Id` = 35188;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (35188, 'ace35188-egglauncher', 35, '2019-02-10 00:00:00') /* Caster */;
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (35188,   1,      32768) /* ItemType - Caster */
+     , (35188,   5,         30) /* EncumbranceVal */
+     , (35188,   9,   16777216) /* ValidLocations - Held */
+     , (35188,  10,   16777216) /* CurrentWieldedLocation - Held */
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:10]:
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+     , (35188,  93,       3092) /* PhysicsState - Ethereal, IgnoreCollisions, Gravity, LightingOn */
+     , (35188,  94,         16) /* TargetType - Creature */
+     , (35188, 106,         10) /* ItemSpellcraft */
+     , (35188, 107,        300) /* ItemCurMana */
+     , (35188, 108,        300) /* ItemMaxMana */
+     , (35188, 117,         10) /* ItemManaCost */
+     , (35188, 151,          2) /* HookType - Wall */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (35188,  22, True ) /* Inscribable */;
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:24]:
```suggestion
VALUES (35188,  22, True ) /* Inscribable */
     , (35188,  23, True ) /* DestroyOnSell */;
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+VALUES (35188,    1,   33560288) /* Setup */
+     , (35188,    3,  536870932) /* SoundTable */
+     , (35188,    6,   67110219) /* PaletteBase */
+     , (35188,    8,  100674625) /* Icon */
+     , (35188,   22,  872415275) /* PhysicsEffectTable */
+     , (35188,   27, 1090519043) /* UseUserAnimation - UseMagicWand */
+     , (35188,   28,       4078) /* Spell - EggBolt */;
+
+
+INSERT INTO `weenie_properties_spell_book` (`object_Id`, `spell`, `probability`)
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:45]:
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+VALUES (35188,  4078,      2) ;
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:46]:
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:47]:
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+INSERT INTO `weenie_properties_palette` (`object_Id`, `sub_Palette_Id`, `offset`, `length`)
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:48]:
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+VALUES (35188, 67114979, 0, 0);
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:49]:
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (35179,   1, 'Egg') /* Name */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (35179,   1,   33554673) /* Setup */
+     , (35179,   3,  536871113) /* SoundTable */
+     , (35179,   8,  100667460) /* Icon */
+     , (35179,  28,       4078) /* Spell - EggBolt */;
+
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql:35]:
```suggestion
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql [hunk 13]
```diff
@@ -0,0 +1,35 @@
+DELETE FROM `weenie` WHERE `class_Id` = 35179;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (35179, 'ace35179-egg', 33, '2019-02-10 00:00:00') /* Missile */;
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql:4]:
```suggestion
VALUES (35179, 'ace35179-egg', 33, '2019-02-10 00:00:00') /* ProjectileSpell */;
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (35179,   1,          0) /* ItemType - None */
+     , (35179,   8,         25) /* Mass */
+     , (35179,   9,          0) /* ValidLocations - None */
+     , (35179,  66,          0) /* CheckpointStatus */
+     , (35179,  93,     133960) /* PhysicsState - ReportCollisions, Missile, AlignPath, PathClipped, LightingOn, Inelastic */;
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35179 Egg.sql:11]:
```suggestion
     , (35179,  93,     133960) /* PhysicsState - Ethereal, IgnoreCollisions, LightingOn, Inelastic */;
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+VALUES (35188,   1, 'Egg Launcher') /* Name */
+     , (35188,  16, 'A nervous hen.') /* LongDesc */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (35188,    1,   33560288) /* Setup */
+     , (35188,    3,  536870932) /* SoundTable */
+     , (35188,    6,   67110219) /* PaletteBase */
+     , (35188,    8,  100674625) /* Icon */
+     , (35188,   22,  872415275) /* PhysicsEffectTable */
+     , (35188,   27, 1090519043) /* UseUserAnimation - UseMagicWand */
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Misc/35188 Egg Launcher.sql:41]:
```suggestion
     , (35188,   27, 1090519043) /* UseUserAnimation - Ready */
```

File: Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Stackable/35179 Egg.sql [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+     , (35179,  79,       0) /* Elasticity */;
+
+INSERT INTO `weenie_properties_string` (`object_Id`, `type`, `value`)
+VALUES (35179,   1, 'Egg') /* Name */;
+
+INSERT INTO `weenie_properties_d_i_d` (`object_Id`, `type`, `value`)
+VALUES (35179,   1,   33554673) /* Setup */
+     , (35179,   3,  536871113) /* SoundTable */
+     , (35179,   8,  100667460) /* Icon */
+     , (35179,  28,       4078) /* Spell - EggBolt */;
```
Comment (LtRipley36706) [inline Database/Patches/2007-03-AncientEnemies/9 WeenieDefaults/Stackable/35179 Egg.sql:34]:
```suggestion
     , (35179,  28,       4078) /* Spell - EggBolt */;

```

**Pull request merged**